views/record_timestamp.go

```diff
@@ -26,5 +26,6 @@ func UpdatedBy(c *ctx.Ctx, updatedAt *time.Time, updatedBy *models.Person, lastU
 	if lastUpdatedBy != nil {
 		s += " Last edit by " + lastUpdatedBy.FullName
 	}
+	s += "."
```

Comment:
Not sure this is right but can't test as this requires a system update to the record. Looks like if there is no last edit (`lastUpdatedBy == nil`), there will be two dots at the end.